specification/azurefleet/AzureFleet.Management/fleet.tsp

```diff
@@ -211,6 +228,274 @@ union EvictionPolicy {
+    """)
+  localStorageDiskTypes?: Array<LocalStorageDiskType>;
+
+  @doc("The range of data disk count specified from Min to Max. Optional parameter. Either Min or Max is required if specified.")
+  dataDiskCount?: VMAttributeMinMaxInteger;
+
+  @doc("The range of network interface count specified from Min to Max. Optional parameter. Either Min or Max is required if specified.")
+  networkInterfaceCount?: VMAttributeMinMaxInteger;
+
+  @doc("The range of network bandwidth in Mbps specified from Min to Max. Optional parameter. Either Min or Max is required if specified.")
+  networkBandwidthInMbps?: VMAttributeMinMaxDouble;
+
+  @doc("Specifies whether the VMSize supporting RDMA (Remote Direct Memory Access) should be used to build Fleet or not.")
+  rdmaSupport?: VMAttributeSupport;
+
+  @doc("The range of RDMA (Remote Direct Memory Access) network interface count specified from Min to Max. Optional parameter. Either Min or Max is required if specified.")
+  rdmaNetworkInterfaceCount?: VMAttributeMinMaxInteger;
+
+  @doc("Specifies whether the VMSize supporting accelerator should be used to build Fleet or not.")
+  acceleratorSupport?: VMAttributeSupport;
+
+  @extension("x-ms-identifiers", [])
+  @doc("The accelerator manufacturers specified as a list. acceleratorSupport should be set to Included or Required to use this VMAttribute.")
+  acceleratorManufacturers?: Array<AcceleratorManufacturer>;
+
+  @extension("x-ms-identifiers", [])
+  @doc("The accelerator types specified as a list. acceleratorSupport should be set to Included or Required to use this VMAttribute.")
+  acceleratorTypes?: Array<AcceleratorType>;
+
+  @doc("The range of accelerator count specified from min to max.. Optional parameter. Either Min or Max is required if specified.")
+  acceleratorCount?: VMAttributeMinMaxInteger;
+
+  @extension("x-ms-identifiers", [])
+  @doc("The VM category specified as a list. Optional parameter.")
+  vmCategories?: Array<VMCategory>;
+
+  @extension("x-ms-identifiers", [])
+  @doc("The VM architecture types specified as a list. Optional parameter.")
+  architectureTypes?: Array<ArchitectureType>;
+
+  @extension("x-ms-identifiers", [])
+  @doc("The VM CPU manufacturers specified as a list. Optional parameter.")
+  cpuManufacturers?: Array<CpuManufacturer>;
+
+  @doc("Specifies whether the VMSize supporting burstable capability should be used to build Fleet or not.")
+  burstableSupport?: VMAttributeSupport;
+
+  @extension("x-ms-identifiers", [])
+  @doc("Specifies which VMSizes should be excluded while building Fleet. Optional parameter.")
+  excludedVMSizes?: Array<string>;
+}
+
+@added(Versions.v2024_11_01)
+@doc("Represents the profile for a single additional location in the Fleet. The location and the virtualMachineProfileOverride (optional).")
+model LocationProfile {
+  @doc("""
+    The ARM location name of the additional region if LocationProfile is specified, then required.
```

Comment:
Nit: Update the comment to this `The ARM location name of the additional region. If LocationProfile is specified, then location is required` or just `The ARM location name of the additional region.`

Reply:
Updated.